packages/keyed/dev/entries.tsx

```diff
@@ -0,0 +1,106 @@
+// changes to this file might be applicable to similar files - grep 95DB7339-BB2A-4F06-A34A-25DDF8BF7AF7
+
+import { createStore, produce } from "solid-js/store";
+import { createEffect } from "solid-js";
+import { Entries } from "../src/index.js";
+import { TransitionGroup } from "solid-transition-group";
+
+const foods = [
+  "oatmeal",
+  "plantains",
+  "cranberries",
+  "chickpeas",
+  "tofu",
+  "Parmesan cheese",
+  "amaretto",
+  "sunflower seeds",
+  "grapes",
+  "vegemite",
+  "pasta",
+  "cider",
+  "chicken",
+  "pinto beans",
+  "bok choy",
+  "sweet peppers",
+  "Cappuccino Latte",
+  "corn",
+  "broccoli",
+  "brussels sprouts",
+  "bread",
+  "milk",
+  "honey",
+  "chips",
+  "cookie",
+];
+const randomIndex = (list: readonly any[]): number => Math.floor(Math.random() * list.length);
+const getRandomFood = () => foods[randomIndex(foods)]!;
+const randomKey = (record: Record<string, string>): string => {
+  const keys = Object.keys(record);
+  return keys[randomIndex(keys)]!;
+};
+
+export default function App() {
+  const [store, setStore] = createStore<{ entries: Record<string, string> }>({
+    entries: {
+      [Math.random()]: "bread",
+      [Math.random()]: "milk",
+      [Math.random()]: "honey",
+      [Math.random()]: "chips",
+      [Math.random()]: "cookie",
+    },
+  });
+
+  const addRandom = () => {
+    setStore("entries", Math.random().toString(), getRandomFood());
+  };
+  const removeRandom = () =>
+    setStore(
+      "entries",
+      produce(p => {
+        delete p[randomKey(p)];
+      }),
+    );
```

Comment:
Woudn't this be the same? it's consistent with changing.
```suggestion
    setStore("entries", p => ({ [randomKey(p)]: undefined }));
```

Reply:
oh duh!